### ISSUE TITLE
Documentation for combined-infracost-comment/README.md might have a config error

### ISSUE THREAD

**Issue by atilasantos** [infracost/infracost-atlantis#81]
Within the bitbucket subsection we should remove the 261 line and put it right after the line 257. In the way it is, all apply workflows will be considered as failed because there'll be no files there when the run is executed.

See images with my tests.
With the config provided in repo
<img width="274" alt="With the config provided in repo" src="https://github.com/infracost/infracost-atlantis/assets/49249082/96c90bb4-3e65-406e-88d1-18e0bfaf4261">
With the config changed
![With the config changed](https://github.com/infracost/infracost-atlantis/assets/49249082/d72832f6-d82f-48f3-87d6-788e0531d159)

**Comment by alikhajeh1** [comment]
@atilasantos thanks! It seems ok to combine them but maybe handle the error case where `infracost comment bitbucket` does an exit 1 and fails; in that case I think the second run command should still run, right?

**Comment by atilasantos** [comment]
Yes, you are right!
We can add a ';' at the end of infracost command and either it is successfully executed or not, the rm command will be executed.

What'd you think?

**Comment by alikhajeh1** [comment]
yeah or `| echo "Always run the next cleanup step even if this command fails"`

**Comment by alikhajeh1** [comment]
@atilasantos I'm curious if you're still using https://github.com/infracost/infracost-atlantis/blob/master/examples/infracost-comment/README.md#running-with-bitbucket ? If so, can you please share your yaml steps so we can review and update that doc if needed?
